src/components/appSwitch/AppSwitch.js

```diff
@@ -226,6 +227,18 @@ const AppSwitch = (props: LoadingProps) => {
     }
   }, [gdstore])
 
+  useEffect(() => {
+    if (showMessageInitAppSwitch) {
+      showErrorDialog('Problem with BlockChain ', '', {
```

Comment:
i'm not sure i understand why we need this generic dialog
each place should have its own customized message and dialog as usuall

Reply:
The message is here because there is an "init()" method

Reply:
the dialog call is here because there is the necessary method "init()"

Reply:
not relevant